demo-shell/src/app/components/app-layout/app-layout.component.ts

```diff
@@ -48,6 +50,25 @@ export class AppLayoutComponent {
         { href: '/about', icon: 'info_outline', title: 'APP_LAYOUT.ABOUT' }
     ];
 
-    constructor() {
+    expandedSidenav: Boolean = false;
```

Comment:
We should avoid using capitalised types, see this:
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html

And when we have an initial value set, type declarations are not needed, type will be inferred.